File: exercicios/R/1149/1149.r
input = file('stdin','r')

line = strsplit(readLines(input,n=1)," ")

a = as.integer(line[[1]][1])
n = as.integer(line[[1]][2])

contv = as.integer(1)

while(n<=0){
    contv = contv+1
    n = as.integer(line[[1]][contv])
}

soma = as.integer(0)

for(i in 1:n-1){
    soma = soma + (a + i)
}
write(soma,'')

File: exercicios/JavaScript/1149/1149.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');

var x = parseInt(line1[0]), n = parseInt(line1[1]), soma=0, count =1;

while(n<=0){
    count += 1;
    n = parseInt(line1[count]);
}

for(let i=0; i<=n -1; i++){
    soma += x + i;
}
console.log(soma);